lib/kernel/host/CMakeLists.txt

```diff
@@ -447,6 +455,8 @@ function(compile_sleef VARIANT SLEEF_CONFIG SLEEF_CONFIG_NEW SLEEF_BC)
 
   endforeach()
 
+  compile_ll_to_bc("${SPIR_WRAPPER_FILE}" "${VARIANT}" BC_FILE_LIST)
+
   file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/${VARIANT}")
   set(SLEEF_BC "${CMAKE_CURRENT_BINARY_DIR}/${VARIANT}/sleef.bc")
   set(SLEEF_BC "${CMAKE_CURRENT_BINARY_DIR}/${VARIANT}/sleef.bc" PARENT_SCOPE)
```

Comment:
Do we want to put the generated wrappers to the repository? I think not if we can always regenerate them using the script.

Reply:
The wrappers are independent of the build host so there's not much point regenerating, but it's true they're quite large. I can remove them and add a custom command to generate it. It adds build dependency on python3 but that should be fine.

Reply:
Right. The benefit is that we ensure the generator doesn't bitrot, stays functional.